gnc/navigator_perception/scan_the_code_detector.cc

```diff
@@ -193,87 +202,104 @@ void ScanTheCodeDetector::init_ros(stringstream& log_msg){
-                                                      left_cam_mat, right_cam_mat);
+    vector<Eigen::Vector3d> position;
+    model_fitter->determine_model_position(position,
+                                           max_features,
+                                           feature_block_size,
+                                           feature_min_distance,
+                                           image_proc_scale,
+                                           diffusion_time,
+                                           current_image_left,
+                                           current_image_right,
+                                           left_cam_mat,
+                                           right_cam_mat);
 }
 
 bool ScanTheCodeDetector::detection_activation_switch(
     sub8_msgs::TBDetectionSwitch::Request &req,
-    sub8_msgs::TBDetectionSwitch::Response &resp) {
+    sub8_msgs::TBDetectionSwitch::Response &resp)
+{
     resp.success = false;
     stringstream ros_log;
     ros_log << "\x1b[1;31mSetting torpedo board detection to: \x1b[1;37m"
-          << (req.detection_switch ? "on" : "off") << "\x1b[0m";
+            << (req.detection_switch ? "on" : "off") << "\x1b[0m";
     ROS_INFO(ros_log.str().c_str());
     active = req.detection_switch;
-    if (active == req.detection_switch) {
-        resp.success = true;
-        return true;
-    }
+    if (active == req.detection_switch)
+        {
+            resp.success = true;
+            return true;
+        }
     return false;
 }
 
 void ScanTheCodeDetector::left_image_callback(
     const sensor_msgs::ImageConstPtr &image_msg_ptr,
-    const sensor_msgs::CameraInfoConstPtr &info_msg_ptr) {
-  left_mtx.lock();
-  model_fitter->left_most_recent.image_msg_ptr = image_msg_ptr;
-  model_fitter->left_most_recent.info_msg_ptr = info_msg_ptr;
-  left_mtx.unlock();
+    const sensor_msgs::CameraInfoConstPtr &info_msg_ptr)
+{
+    left_mtx.lock();
+    left_most_recent.image_msg_ptr = image_msg_ptr;
+    left_most_recent.info_msg_ptr = info_msg_ptr;
+    left_mtx.unlock();
 }
 
 void ScanTheCodeDetector::right_image_callback(
     const sensor_msgs::ImageConstPtr &image_msg_ptr,
-    const sensor_msgs::CameraInfoConstPtr &info_msg_ptr) {
-  right_mtx.lock();
-  model_fitter->right_most_recent.image_msg_ptr = image_msg_ptr;
-  model_fitter->right_most_recent.info_msg_ptr = info_msg_ptr;
-  right_mtx.unlock();
+    const sensor_msgs::CameraInfoConstPtr &info_msg_ptr)
+{
+    right_mtx.lock();
+    right_most_recent.image_msg_ptr = image_msg_ptr;
+    right_most_recent.info_msg_ptr = info_msg_ptr;
+    right_mtx.unlock();
 }
 
 
-int main(int argc, char **argv) {
-  ros::init(argc, argv, "scan_the_code_board_perception");
-  ROS_INFO("Initializing node /scan_the_code_board_perception");
-  ScanTheCodeDetector scan_the_code_detector;
-  ros::spin();
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "scan_the_code_board_perception");
```

Comment:
scan_code_perception maybe? We don't want node names to be so long unnecessarily.